lib/Config.ts

```diff
@@ -14,6 +14,18 @@ import type { ClnConfig } from './lightning/cln/Types';
 import type { SidecarConfig } from './sidecar/Sidecar';
 import type { NodeSwitchConfig } from './swap/NodeSwitch';
 
+type EmailConfig = {
+  enabled: boolean;
+  host?: string;
+  port?: number;
+  secure?: boolean; // true for 465, false for other ports
```

Comment:
```suggestion
  secure?: boolean;
```

That's just for your specific provider